### PR TITLE
Ilcm

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: faqs.md [hunk 1]
```diff
@@ -2,6 +2,26 @@
...
+
+- Built-in WOG compliance and best practices as baselines.
+- Out-of-the-box integrations with TechPass. In addition, in the future we also intend to enable default integration with some of the WOG services such as GCSOC reducing the overheads cost and additional processes for tenants.
+- Community such as InnerSource integration for sharing and adopting WOG requirements. Following are some of the features offered as a Community:
+  - SIEM Framework
+  - (Upcoming) Visualisation Framework
+  - (Upcoming) Ingestions
+  - (Upcoming) Configurations
+  - (Upcoming) Infrastructure-as-Code (IaC)
+- (Upcoming) Designed for collaboration across projects within StackOps
```
Comment (o20ne) [inline faqs.md:18]: Let's remove "Designed for"
- (Upcoming) Collaboration across projects within StackOps
would be better

File: faqs.md [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+- Enhanced Elastic Common Schema to improve Data identity, Data profiling and Data MarketPlace
```
Comment (o20ne) [inline faqs.md:19]: - Enhanced Elastic Common Schema to improve Data identity, Data profiling and enables Data MarketPlace

File: faqs.md [hunk 3]
```diff
@@ -17,33 +37,21 @@ Yes, StackOps deployments are compatible with Elastic plugins and adaptors.
...
-
+- We offer Built-in compliance and best practices as baselines to ensure smooth and efficient operations.
+- Out-of-box integrations with TechPass and other future Whole-of-Government (WoG) required services.
+- You will be part of a community such as InnerSource integration where you can connect with peers, share knowledge, and collectively address WOG needs.
+    - Built-in SIEM rules
+    - Visualisation features (upcoming)
+    - Infrastructure as Code capabilities (upcoming) and
+    - Other features
+- StackOps is designed to facilitate collaboration leading to a seamless teamwork
+- We constantly strive to stay up to date with the evolution of technology, including advancements related to competency frameworks such as OpenTelemetry.
```
Comment (o20ne) [inline faqs.md:52]: Let's remove "related to competency frameworks"

**Review by o20ne — CHANGES_REQUESTED** [review]

File: faqs.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply faqs.md:18]: done

File: faqs.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply faqs.md:19]: Does improving Data Identity and Data profiling enable Data Marketplace?

Comment (SenthilKumarSivakami) [reply faqs.md:19]: done

File: faqs.md (hunk 3 above)
Comment (SenthilKumarSivakami) [reply faqs.md:52]: done, this info was duplicated in 2 places so have removed the redundant instance. Thanks!

**Review by paulkarippa — APPROVED** [review]

**Review by o20ne — APPROVED** [review]

**Pull request merged**